digdag-docs/src/conf.py

```diff
@@ -63,7 +63,7 @@ def setup(app):
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
```

Comment:
Seems that this keeps copying html and yaml file into `_static` directory under `build/html` on my local environment. Is it as you expected?

Reply:
Tried to use `html_extra_path` but it can't copy directories.

Reply:
@frsyuki @yoyama  Finally, I got way to bundle `api` directory setting as:

```suggestion
html_static_path = ['_static']
html_extra_path = ['_extra']
```

with creating `src/_extra` and move current `src/api` directory under `src/_extra/` like:

```
$ tree src/_extra
src/_extra
└── api
    ├── index.html
    └── swagger.yaml
```